### DISCUSSION TITLE
Limit media uploads per user to directory?

### DISCUSSION THREAD

**Discussion by HeartlandTechie** [area17/twill#1068] [Q&A]
Is there any handy way I could have Twill create media in a directory per user?  So let's say I have a user associated with a business and I just want their photos to show up and for them to manipulate just their photos?   Thoughts?

**Comment by pboivin** [comment]
Hey @HeartlandTechie, I don't think this is possible without heavy customizing at the moment. Twill currently has no concept of who uploaded what media.
